data/libs/systems/vip.lua

```diff
@@ -52,16 +53,15 @@ end
 
 function CheckPremiumAndPrint(player, msgType)
 	if player:getVipDays() == 0xFFFF then
-		player:sendTextMessage(msgType, "You have infinite amount of VIP days left.")
+		player:sendTextMessage(msgType, config.vipDaysInfiniteMessage)
```

Comment:
As this message is only displayed once, I think it needs to be used in a variable.